scripts/table_updates/utilities.py

```diff
@@ -212,8 +212,71 @@ def remove_backslash(df: pandas.DataFrame, cols: List[str]) -> pandas.DataFrame:
         raise ValueError("Invalid column list. Not all columns are in the dataframe.")
 
 
+def extract_site_name_from_sample_id(sample_id: str) -> str:
+    """Extract site name from CPT Genie Sample ID
+       Site name is the second substring of the sample ID
+
+    Args:
+        cpt_genie_sample_id (str): sample ID
+
+    Returns:
+        str: Site name
+    """
+    return sample_id.split("-")[1]
+
+
+def convert_mapping_table_to_long(
```

Comment:
nit: for consistency, function can match other tier1a functions like `convert_tier1a_data_replacement_mapping_table_to_long()`